lib/snippets/publish-lerna-independent-packages

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env node
+const {resolve} = require('path');
+const {execSync} = require('child_process')
+const Repository = require(resolve('.', 'node_modules', 'lerna', 'lib', 'Repository'));
+const PackageUtilities = require(resolve('.', 'node_modules', 'lerna', 'lib', 'PackageUtilities'));
+
+function npmTag(version) {
+  const isNext = ['-beta', '-alpha', '-rc', '-next'].some(distributionType =>
```

Comment:
This seems wrong, it wouldn't catching things like `1.0.0-beta.1` would it? I think maybe we just want to match any version that is not strictly `/\d+\.\d+\.\d+/` and assume that's not meant to be a `latest`-tagged version.

Reply:
Ah, yeah, yeah.  We can't use the strict regexp, because we have to support stuff like `@shopify/tinymce@4.5.5-shopify5` as latest.

I've updated the logic to use `includes`, which matches the [ruby equivalent](https://github.com/Shopify/shipit-engine/blob/master/app/models/shipit/deploy_spec/npm_discovery.rb#L50).  LMK if that makes sense to you, please.

Reply:
👍